frontend/messages/de.json

```diff
@@ -2,9 +2,19 @@
 	"$schema": "https://inlang.com/schema/inlang-message-format",
 	"help": "Hilfe",
 	"menuDisplayOnMap": "Auf der Karte anzeigen",
+	"menuOverlay": "Überlagerung",
```

Comment:
Then also "Überlagerung auf der Karte anzeigen" for next line

Reply:
In German everybody understands also 'Overlay'. If you want...